target/linux/octeon/image/Makefile

```diff
@@ -23,22 +23,24 @@ define Build/strip-kernel
 endef
 
 define Device/generic
-  FILESYSTEMS := ext4
   DEVICE_TITLE := Generic
+  FILESYSTEMS := ext4
```

Comment:
Why is this showing up as diff here?

Reply:
I will fix it.

Reply:
Just realized that my question might sound rude from a different angle, that was not my intention.
Thanks for fixing.

Reply:
It did not sound rude. Maybe I made the impression? All fine. Let's keep hacking. :-)